Dockerfile

```diff
@@ -27,6 +27,10 @@ ARG BASE_TAG
 ARG BASE_IMAGE
 ARG LAUNCHER
 
+ENV PATH="/usr/local/cuda-10.2/bin:${PATH}"
```

Comment:
You shouldn't set the PATH in the main docker file, as this will mess up the AMD64 branch. This PATH and LD_LIBRARY_PATH should only be set in the arch-specific file.

Consult @afdaniele for a more detailed explanation.

Reply:
Alternatively, if you have to specify PATH and LD_LIBRARY_PATH, you should append both 10.1 and 10.2 together so the 10.1 (AMD64) is not looking for 10.2 cuda

Reply:
Usually if I'm not wrong it's enough to link the cuda folder. This cuda folder should be a sort of link I guess to the correct version installed on the system. But I didn't find the folder cuda on the image. I check again and I'll correct it.